opentracing/src/main/java/opentracing/Span.java

```diff
@@ -65,29 +63,7 @@
   Span setTag(String key, Number value);
 
   /**
-   * {@code message} is a format string and can refer to fields in the payload by path, like so:
-   *
-   * <pre>{@code
-   *
-   * span.info("first transaction is worth ${transactions[0].amount} ${transactions[0].currency}",
-   *     ImmutableMap.of(
-   *       "transactions", asList(
-   *         Transaction.builder().amount(10).currency("USD").build(),
-   *         Transaction.builder().amount(11).currency("USD").build(),
-   *       )
-   *     )
-   * );
-   * }</pre>
-   *
-   * @param message {@link Formatter format string} that can refer to fields in the args payload.
-   * @param args arbitrary payload
-   */
-  // See https://github.com/opentracing/opentracing.github.io/issues/30 about parameterization
-  Span info(String message, Object... args);
-
-  /** Same as {@link #info}, but for warnings. */
-  Span warning(String message, Object... args);
-
-  /** Same as {@link #info}, but for errors. */
-  Span error(String message, Object... args);
+   * @todo
+   **/
+  Span event(String message, /* @Nullable */ Object payload);
```

Comment:
I am too out-of-the-loop in Java to justify strong feelings about naming/spelling. That said, it is important that we have a version of this method that supports an explicit timestamp (for historical recording, a la https://github.com/opentracing/opentracing.github.io/issues/22).

FWIW, I personally prefer `log` as the catch-all "plumbing" name and `logEvent` for the slightly easier-to-use event-oriented API. I'm also fine only-supporting `log` for now. Per https://github.com/opentracing/opentracing.github.io/issues/30, the idea is that we'll come back and deal with logging porcelain once the more important (and far more subtle) propagation APIs are settled.